.circleci/config.yml

```diff
@@ -100,6 +100,17 @@ jobs:
           name: Deployment
           command: sh ./gradle/deploy.sh
 
+  stackdriver-it:
+    executor: circle-jdk-executor
+    steps:
+      - run:
+          name: Setup Google Cloud auth
+          command: |
+            echo "$GCLOUD_SERVICE_KEY" >> "$HOME"/gcloud-service-key.json
+            echo 'export GOOGLE_APPLICATION_CREDENTIALS="$HOME"/gcloud-service-key.json' >> "$BASH_ENV"
```

Comment:
This seems to have a default location so that: `GOOGLE_APPLICATION_CREDENTIALS` is not needed (but in that case you need to create the folder with `mkdir -p`):
```
$HOME/.config/gcloud/application_default_credentials.json
```

Reply:
It's probably better to not rely on the default, and since it wouldn't eliminate the need to have a script, I think we can keep it this way. I was aligning with the CircleCI GCP orb defaults because I thought that might make things easier for us, but in the end it didn't so I removed it.